features/enable_fips_pro.feature

```diff
@@ -12,11 +12,15 @@ Feature: FIPS enablement in PRO cloud based machines
         Then I verify that `fips` is disabled
         And I verify that `fips-updates` is disabled
         When I run `pro enable <fips-service> --assume-yes` with sudo
-        Then stdout contains substring:
+        Then stdout matches regexp:
         """
         Updating <fips-name> package lists
         Installing <fips-name> packages
-        Updating standard Ubuntu package lists
+        This will downgrade the kernel from .+ to .+\.
```

Comment:
Is this test supposed to pass on all releases specified in this `Scenario Outline`  ? I have tested the `bionic` `aws.pro` and the test didn't work

Reply:
Where is this failing for you?  It worked for all the iterations I ran of it and is currently passing in CI -- the aws.pro bionic failure is only for an unrelated proxy test.

Reply:
That's weird. I am getting the following error on bionic:
```
      Assertion Failed: Expected to match regexp:                                                                                                                                                                  
        Updating FIPS package lists                                                                                                                                                                                
        Installing FIPS packages                                                                                                                                                                                   
        This will downgrade the kernel from .+ to .+\.                                                                                                                                                             
        Warning: Downgrading the kernel may cause hardware failures.  Please ensure the                                                                                                                            
                 hardware is compatible with the new kernel version before proceeding.                                                                                                                             
                                                                                                                                                                                                                   
        Updating standard Ubuntu package lists(\n.*)?                                                                                                                                                              
        FIPS enabled                                                                                                                                                                                               
        A reboot is required to complete install                                                                                                                                                                   
      But got:                                                                                                                                                                                                     
        One moment, checking your subscription first                                                                                                                                                               
        Disabling incompatible service: Livepatch                                                                                                                                                                  
        Updating FIPS package lists                                                                                                                                                                                
        Installing FIPS packages                                                                                                                                                                                   
        Updating standard Ubuntu package lists                                                                                                                                                                     
        FIPS enabled                                                                                                                                                                                               
        A reboot is required to complete install.  
```

But I haven't checked the other releases yet

Reply:
I think I see the issue.  Checking the AWS listings it looks like Canonical only has 16.04 Pro FIPS (not plain pro) so the kernel will not be being downgraded in that instance.  I went back to check the logs in the CI run and it seems this test is getting skipped there.  Not sure why that is happening when it runs locally.

Reply:
Hi, thanks for the check!  I have since fixed the downgrade conditional -- let me know if you're still seeing unexpected failures?